### DISCUSSION TITLE
Parsing out values from WHERE and removing it from WHERE clause

### DISCUSSION THREAD

**Discussion by AStvnMC** [JSQLParser/JSqlParser#1577] [General]
Hello, 
I was wondering if there is a way to remove some target strings from the WHERE clause and create a new SELECT that does not contain the target strings.

For Example:
SELECT * FROM table WHERE date > 0 AND start = '2022' AND end = '2023';

Wanted Output:
SELECT * FROM table WHERE date > 0;

And I would want to be able to obtain the start and end values and store it myself.

**Comment by AStvnMC** [comment]
Bump

**Comment by wumpz** [comment]
Maybe this helps: https://stackoverflow.com/questions/57055443/how-to-add-where-condition-to-sql-with-jsqlparser. The parser gives you a statement object you could traverse by your own. So dependent of the structure of this where statement you can naturally read out those start / end values.

**Reply by AStvnMC** [reply]
Thanks I did end up traversing the tree haha
